src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -281,12 +290,13 @@ protected function getConstructor(array &$data, $class, array &$context, \Reflec
      * @param array            $context
      * @param \ReflectionClass $reflectionClass
      * @param array|bool       $allowedAttributes
+     * @param string|null      $format
      *
      * @return object
      *
      * @throws RuntimeException
      */
-    protected function instantiateObject(array &$data, $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes)
+    protected function instantiateComplexObject(array &$data, $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes, $format = null)
```

Comment:
What do you think about using ``instantiateClass`` to have something shorter but still clear ?